test/externalTests/common.sh

```diff
@@ -189,20 +189,30 @@ function run_test
 
     force_solc "$CONFIG" "$DIR" "$SOLJSON"
 
-    printLog "Checking optimizer level..."
-    if [ -z "$OPTIMIZER_LEVEL" ]; then
-        printError "Optimizer level not found. Please define OPTIMIZER_LEVEL=[1, 2, 3]"
+    printLog "Checking optimizer levels..."
+    if [ -z "$OPTIMIZER_LEVELS" ]; then
+        printError "Optimizer level not found. Please define OPTIMIZER_LEVELS=[0, 1, 2, 3, 4, 5]"
```

Comment:
old -> new (renaming)
1 -> 5
2 -> 3
3 -> 2

Newly added levels
0 -> legacy optimizer disabled
1 -> legacy optimizer enabled
4 -> legacy optimizer disabled, new optimizer enabled

Did I get that right?

Reply:
Yes, that's exactly what it means :)